[2024-11-14T07:09:29.171+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-14T07:09:29.651+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: model_training_update.update_data manual__2024-11-14T07:09:21.219651+00:00 [queued]>
[2024-11-14T07:09:29.763+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: model_training_update.update_data manual__2024-11-14T07:09:21.219651+00:00 [queued]>
[2024-11-14T07:09:29.771+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-11-14T07:09:29.997+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): update_data> on 2024-11-14 07:09:21.219651+00:00
[2024-11-14T07:09:30.314+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=300) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-11-14T07:09:30.334+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'model_training_update', 'update_data', 'manual__2024-11-14T07:09:21.219651+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmpn_ls918i']
[2024-11-14T07:09:30.368+0000] {standard_task_runner.py:72} INFO - Started process 302 to run task
[2024-11-14T07:09:30.369+0000] {standard_task_runner.py:105} INFO - Job 7: Subtask update_data
[2024-11-14T07:09:30.648+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-11-14T07:09:31.792+0000] {task_command.py:467} INFO - Running <TaskInstance: model_training_update.update_data manual__2024-11-14T07:09:21.219651+00:00 [running]> on host 61d1a0d6a190
[2024-11-14T07:09:33.297+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='model_training_update' AIRFLOW_CTX_TASK_ID='update_data' AIRFLOW_CTX_EXECUTION_DATE='2024-11-14T07:09:21.219651+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-14T07:09:21.219651+00:00'
[2024-11-14T07:09:33.321+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-14T07:09:35.245+0000] {python.py:240} INFO - Done. Returned value was: {'message': 'Données mises à jour avec succès.'}
[2024-11-14T07:09:36.185+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-14T07:09:36.205+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=model_training_update, task_id=update_data, run_id=manual__2024-11-14T07:09:21.219651+00:00, execution_date=20241114T070921, start_date=20241114T070929, end_date=20241114T070936
[2024-11-14T07:09:36.735+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-14T07:09:37.758+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-14T07:09:37.870+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
